src/main/deploy/swerve/modules/physicalproperties.json

```diff
@@ -2,7 +2,7 @@
   "optimalVoltage": 12,
   "wheelGripCoefficientOfFriction": 1.19,
   "currentLimit": {
-    "drive": 60,
+    "drive": 140,
```

Comment:
Should drop this to 120, as was consistently reaching brownout conditions near the end of a match. Can do after the PR.